cmd/minikube/cmd/start_flags.go

```diff
@@ -253,17 +253,14 @@ func generateClusterConfig(cmd *cobra.Command, existing *config.ClusterConfig, k
 		klog.Info("no existing cluster config was found, will generate one from the flags ")
 		cc = generateNewConfigFromFlags(cmd, k8sVersion, drvName)
 
-		cnm, err := cni.New(cc)
+		cnm, err := cni.New(&cc)
```

Comment:
why ?

Reply:
this is to be able to set kubelet.cni-conf-dir via cc.KubernetesConfig.ExtraOptions in cni.configureCNI if needed